packages/integrations/image/src/endpoint.ts

```diff
@@ -14,7 +14,8 @@ async function loadRemoteImage(src: URL) {
 		}
 
 		return Buffer.from(await res.arrayBuffer());
-	} catch {
+	} catch (err: unknown) {
+		console.error(err)
```

Comment:
I think we should do the same at https://github.com/withastro/astro/blob/eab4a0279fd28163cea9da2b010c5ccde5645e6f/packages/integrations/image/src/build/ssg.ts#L55